index.js

```diff
@@ -154,10 +154,22 @@ function packAndInstallLibrary(name, dir, targetDir) {
     fs.mkdirSync(libDestDir, { recursive: true })
 
     const tmpName = name.replace(/[\s\/]/g, "-").replace(/@/g, "")
+
+   
+
     // npm replaces @... with at- where yarn just removes it, so we test for both files here
     const regex = new RegExp(`^(at-)?${tmpName}(.*).tgz$`)
 
-    const packagedName = fs.readdirSync(dir).find(file => regex.test(file))
+    let packagedName = fs.readdirSync(dir).find(file => regex.test(file))
+
+    if(hasYarn) {
+      let yarnVersion = spawn.sync(['-v'], { cwd: dir, encoding: "utf8" });
```

Comment:
Isn't there a longhand version of the same flag, which will clearly signal the intent of the command? I know that yarn@1 has `--version` in addition to `-v`.